lib/ui/map/map_page/map_page.dart

```diff
@@ -513,6 +516,35 @@ class _MapPageState extends PageStatePlante<MapPage>
     });
   }
 
+  Widget _timedHints() {
```

Comment:
nit: I would extract this code into a separate widget, just like `MapPageProgressBar` wraps progresses for map's progressbar. It's not something 100% necessary, but it would make `map_page.dart` a little bit easier to understand.

Reply:
yes, will do